deployment/terraform/output.go

```diff
@@ -87,11 +87,40 @@ type Output struct {
 
 // Instance is an AWS EC2 instance resource.
 type Instance struct {
-	PrivateIP  string `json:"private_ip"`
-	PublicIP   string `json:"public_ip"`
-	PublicDNS  string `json:"public_dns"`
-	PrivateDNS string `json:"private_dns"`
-	Tags       Tags   `json:"tags"`
+	PrivateIP      string `json:"private_ip"`
+	PublicIP       string `json:"public_ip"`
+	PublicDNS      string `json:"public_dns"`
+	PrivateDNS     string `json:"private_dns"`
```

Comment:
Let's make these private fields to prevent mistakes of accessing them in the future.

Reply:
I realized that making these fields private would stop JSON unmarshal. In that case, I'd suggest to have a separate struct to unmarshal `InstanceToParse` and then create a new `Instance` by calling the Set methods.

Reply:
I don't think this is necessary, there may be situations where we want to use the private or public ips directly. For example for intra-server communication we favor Private IPs over Public ones. One example is on the proxy IP in the `getServerURL` method. I'm currently reviewing usages and making sure I'm not breaking the services. Will re-request review once done and we can discuss in there, there is some tooling that I don't know how it would behave with this change.

Reply:
> For example for intra-server communication we favor Private IPs over Public ones. One example is on the proxy IP in the getServerURL method. I'm currently reviewing usages and making sure I'm not breaking the services. 

Ok, this is tricky. I'd prefer to have clear APIs that cannot be misused.